stable/insights-agent/templates/kube-bench/cronjob.yaml

```diff
@@ -0,0 +1,100 @@
+{{- define "kubebench.job.spec" -}}
+{{- $_ := set . "Label" "kube-bench" }}
+{{- $_ := set . "Config" .Values.kubebench }}
+restartPolicy: Never
+serviceAccountName: {{ include "insights-agent.fullname" . }}
+{{- if or (eq .Values.kubebench.deploymentMode "daemonsetMaster") (eq .Values.kubebench.deploymentMode "daemonset") }}
+volumes:
+- name: output
+  emptyDir: {}
+containers:
+- name: kube-bench-aggregator
+  image: "{{ .Values.kubebench.aggregator.image.repository }}:{{ .Values.kubebench.aggregator.image.tag }}"
+  imagePullPolicy: Always
+  env:
+  - name: DAEMONSET_SERVICE
+    value: {{ include "insights-agent.fullname" . }}-kube-bench-svc
+  resources:
+    {{- toYaml .Values.kubebench.aggregator.resources | nindent 4 }}
+  volumeMounts:
+  - name: output
+    mountPath: /output
+  securityContext:
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+      - ALL
+    privileged: false
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+{{- else }}
+hostPID: true
+volumes:
+- name: output
+  emptyDir: {}
+- name: tmp
+  emptyDir: {}
+- name: var-lib
+  hostPath:
+    path: "/var/lib"
+    type: Directory
+- name: etc-systemd
+  hostPath:
+    path: "/etc/systemd"
+    type: Directory
+- name: etc-kubernetes
+  hostPath:
+    path: "/etc/kubernetes"
+    type: Directory
+- name: usr-bin
+  hostPath:
+    path: "/usr/bin"
+    type: Directory
+containers:
+- image: "{{ .Values.kubebench.image.repository }}:{{ .Values.kubebench.image.tag }}"
```

Comment:
Can we factor out this container? Looks like it's mostly (entirely?) similar between cronjob and daemonset